packages/developer-console-spa/src/components/AppSettings.tsx

```diff
@@ -63,11 +62,11 @@ function AppSettings () {
     event.preventDefault();
     deleteAppService( app.id )
       .then( () => {
-        window.OpNotification.success( { subject: 'App Deleted Succssfully!' } );
+        window.OpNotification?.success( { subject: 'App Deleted Successfully!' } );
         history.push( '/' );
       } )
       .catch( err => {
-        window.OpNotification.danger( { subject: 'An error occured when deleting the App.', body: 'Please try again later.' } );
+        window.OpNotification?.danger( { subject: 'An error occured when deleting the App.', body: 'Please try again later.' } );
```

Comment:
```suggestion
        window.OpNotification?.danger( { subject: 'An error occurred when deleting the App.', body: 'Please try again later.' } );
```

Reply:
Oh boy, I need to be more careful😅. My keyboard keeps missing some keystrokes.
Updated the commit.